govcd/api_vcd_versions.go

```diff
@@ -26,6 +26,27 @@ type SupportedVersions struct {
 	VersionInfos `xml:"VersionInfo"`
 }
 
+// apiVersionToVcdVersion gets the vCD version from max supported API version
+var apiVersionToVcdVersion = map[string]string{
+	"29.0": "9.0",
+	"30.0": "9.1",
+	"31.0": "9.5",
+	"32.0": "9.7",
+	"33.0": "10.0",
+}
+
+// vcdVersionToApiVersion gets the max supported API version from vCD version
+var vcdVersionToApiVersion = map[string]string{
+	"9.0":  "29.0",
+	"9.1":  "30.0",
+	"9.5":  "31.0",
+	"9.7":  "32.0",
+	"10.0": "33.0",
+}
+
+// to make apiVersionToVcdVersion used
+var _ = vcdVersionToApiVersion
```

Comment:
The above comment uses the old name. BTW, would you extended the comment to say _why_ does it need to be used? Also, why only one of these maps gets this treatment, but not both?

Reply:
Comment fixed. This map also was added as it was requested to have ready maps to convert API to vCD version and versa. This variable initialization is needed as for a moment that map isn't used.